components/tournament_ticker/tournament_ticker.lua

```diff
@@ -69,9 +84,15 @@ local STATUS_ONGOING = {
 local STATUS_CONCLUDED = {
 	enum = TOURNAMENT_STATUS.FINISHED,
 	isTournamentInStatus = function(tournament)
-		--- TODO return false if after threshold
-		return tournament.startDate.timestamp < DateExt.getCurrentTimestamp()
-			and tournament.endDate.timestamp > DateExt.getCurrentTimestamp()
+		-- No known enddate, cannot have finished
+		if tournament.endDate.timestamp == DateExt.defaultTimestamp then
+			return false
+		end
+		-- Has ended
+		if DateExt.getCurrentTimestamp() >= tournament.endDate.timestamp then
+			return false
+		end
+		return true
```

Comment:
option1:
```suggestion
		-- Has not ended
		if DateExt.getCurrentTimestamp() < tournament.endDate.timestamp then
			return false
		end
		return true
```
option2:
```suggestion
		-- Has ended
		if DateExt.getCurrentTimestamp() >= tournament.endDate.timestamp then
			return true
		end
		return false
```

also as mentioned in the comment above again i suggest to shift by 1 day here too